# isup.c:2099:17: error: no member named 'st_mtimespec' in 'struct stat'

--- a/src/isup.c
+++ b/src/isup.c
@@ -2081,7 +2081,7 @@
 static time_t now_t;
 static char *cacheFile, *cacheLock, *cacheControl;
 
-#if defined(__APPLE__) || defined(__ANDROID__)
+#if defined(__APPLE__)
 #define ns_member st_mtimespec
 #else
 #define ns_member st_mtim
